packages/app/src/pages/Rewards/Active/index.tsx

```diff
@@ -0,0 +1,220 @@
+  const monthlyReward = annualReward / 12 || 0
+  const dailyReward = annualReward / 365 || 0
+
+  const handleToggleStake = () => {
+    if (isCustomStake) {
+      setManualStake(null)
+    }
+    setIsCustomStake(!isCustomStake)
+  }
+
+  return (
+    <>
+      <StatRow>
+        <RewardText>
+          <Text
+            label={t('rewards.averageRewardRate')}
+            value={`${rewardRate.toFixed(2)}%`}
+            helpKey="Average Reward Rate"
+          />
+        </RewardText>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            gap: '1rem',
+          }}
+        >
+          <Text
+            label={
+              isCustomStake
+                ? t('rewards.customBalance')
+                : t('rewards.currentStakedBalance')
+            }
+            value={`${currentStake.toLocaleString('en-US', {
+              minimumFractionDigits: 2,
+              maximumFractionDigits: 2,
+            })} ${networkData.api.unit}`}
+            helpKey={isCustomStake ? 'Custom Balance' : 'Your Balance'}
+          />
+          <ButtonPrimary
+            text={
+              isCustomStake
+                ? t('rewards.useConnectedWallet')
+                : t('rewards.useCustomAmount')
+            }
+            onClick={handleToggleStake}
+            style={{ minWidth: 'fit-content', whiteSpace: 'nowrap' }}
+          />
+        </div>
+      </StatRow>
+
+      <AnimatePresence>
+        {isCustomStake && (
+          <PageRow>
+            <motion.div
+              initial={{ opacity: 0, height: 0, marginTop: 0 }}
+              animate={{ opacity: 1, height: 'auto', marginTop: '1rem' }}
+              exit={{ opacity: 0, height: 0, marginTop: 0 }}
+              transition={{
+                duration: 0.3,
+                ease: 'easeInOut',
+              }}
+              style={{ overflow: 'hidden' }}
+            >
+              <CardWrapper>
+                <CardHeader>
+                  <h4>{t('rewards.adjustStake')}</h4>
+                </CardHeader>
+
+                <div style={{ padding: '1rem' }}>
+                  <label htmlFor="manual-stake">
+                    {t('rewards.enterStakeAmount')} ({networkData.api.unit}):
+                  </label>
+                  <input
+                    id="manual-stake"
+                    type="number"
+                    step="0.01"
+                    value={manualStake ?? ''}
+                    onChange={(e) =>
+                      setManualStake(Number(e.target.value) || null)
+                    }
+                    placeholder={t('rewards.stakePlaceholder')}
+                    style={{
+                      width: '100%',
+                      padding: '0.5rem',
+                      marginTop: '0.5rem',
+                      border: '1px solid var(--border-primary-color)',
+                      borderRadius: '4px',
```

Comment:
Let's use a `rem` value here so all styling is in proportion to the global font size.

```suggestion
                      borderRadius: '0.5rem',
```